back-end/legeno-around-here/src/test/java/wooteco/team/ittabi/legenoaroundhere/acceptance/AwardAcceptanceTest.java

```diff
@@ -23,9 +23,8 @@
     @BeforeEach
     void setUp() {
         RestAssured.port = port;
-        String location = createUser(TEST_USER_EMAIL, TEST_USER_NICKNAME, TEST_USER_PASSWORD);
-        userId = getIdFromUrl(location);
         TokenResponse token = login(TEST_USER_EMAIL, TEST_USER_PASSWORD);
+        userId = TEST_USER_ID;
```

Comment:
별 건 아니지만, 순서를 바꾸는게 더 읽기 편한 것 같아요!

```suggestion
        userId = TEST_USER_ID;
        TokenResponse token = login(TEST_USER_EMAIL, TEST_USER_PASSWORD);
```